Classes/Job/JobManager.php

```diff
@@ -127,23 +127,23 @@ public function waitAndExecute(string $queueName, $timeout = null): ?Message
             } else {
                 $this->executeJobForMessage($queue, $message);
             }
-        } catch (\Exception $exception) {
+        } catch (\Throwable $throwable) {
             $maximumNumberOfReleases = isset($queueSettings['maximumNumberOfReleases']) ?
                 (int)$queueSettings['maximumNumberOfReleases'] :
                 self::DEFAULT_MAXIMUM_NUMBER_RELEASES;
             if ($message->getNumberOfReleases() < $maximumNumberOfReleases) {
                 $releaseOptions = isset($queueSettings['releaseOptions']) ? $queueSettings['releaseOptions'] : [];
                 $queue->release($message->getIdentifier(), $releaseOptions);
-                $this->emitMessageReleased($queue, $message, $releaseOptions, $exception);
-                $logMessage = $this->throwableStorage->logThrowable($exception);
+                $this->emitMessageReleased($queue, $message, $releaseOptions, $throwable);
```

Comment:
Doh, this part now fails if `$throwable` is an `\Error` – and changing the signal signature would be a breaking change

Reply:
😬